src/editor/files/FileChooser.java

```diff
@@ -0,0 +1,82 @@
+package editor.files;
+
+import editor.ui.TextEditor;
+
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import javax.swing.filechooser.FileSystemView;
+
+/**
+ *  Class for selects files via JFileChooser UI element for open and save it
+ *
+ * @see FileLoader - opens and saves files
+ */
+public class FileChooser {
+
+    /**
+     * This method opens file
+     */
+    public static void open() {
+        JFileChooser fileChooser;
+        String fileContent;
+        /*
+           Try to use last path, or start with default one
+         */
+        String path = FileLoader.loadFile("path.txt");
+        if (path != null){
+            fileChooser = new JFileChooser(path);
+        } else {
+            fileChooser = new JFileChooser((FileSystemView.getFileSystemView().getHomeDirectory()));
+        }
+
+        fileChooser.setDialogTitle("Select text file");
+        FileNameExtensionFilter filter = new FileNameExtensionFilter("Text files","mytxt");
+        fileChooser.addChoosableFileFilter(filter);
+        fileChooser.setAcceptAllFileFilterUsed(true);
+
+        if (fileChooser.showOpenDialog(null) == JFileChooser.APPROVE_OPTION) {
+            /*
+                Remember new start path
+             */
+            path = fileChooser.getSelectedFile().getParent();
+            FileLoader.saveFile("path.txt", path, false);
+            /*
+                Load contents from file
+             */
+            fileContent = FileLoader.loadFile(fileChooser.getSelectedFile().getAbsolutePath());
+            TextEditor.setFileName(fileChooser.getSelectedFile().getAbsolutePath());
+            TextEditor.getTextArea().setText(fileContent);
+        }
+    }
+
+    /**
+     * This metod saves the file
+     * @return path to it
+     */
+    static String save() {
+        JFileChooser fileChooser;
+        /*
+           Try to use last path, or start with default one
+         */
+        String path = FileLoader.loadFile("path.txt");
+        if (path != null){
+            fileChooser = new JFileChooser(path);
+        } else {
+            fileChooser = new JFileChooser((FileSystemView.getFileSystemView().getHomeDirectory()));
+        }
+
+        fileChooser.setDialogTitle("Save text file");
+        fileChooser.setDialogType(JFileChooser.SAVE_DIALOG);
+
+        if (fileChooser.showSaveDialog(null) == JFileChooser.APPROVE_OPTION){
+            /*
+                Remember new start path
```

Comment:
A lot of comments is not always good. If you need a comment inside your code it means the code is not very self-descriptive. In general, comments are good:
- to describe an API like (JavaDoc comments) /**...  */ for classes and methods
- comments within the code only for hard or unobvious places.

If the code is clear enough it doesn't need comments. So, one solution is decomposed your methods into smaller one with human-readable names. Then people could read your code like a book without extra comments.